packages/@aws-cdk/aws-redshift-alpha/README.md

```diff
@@ -556,6 +556,23 @@ const cluster = new Cluster(this, 'Redshift', {
 cluster.addIamRole(role);
 ```
 
+## [Multi-AZ](https://docs.aws.amazon.com/redshift/latest/mgmt/managing-cluster-multi-az.html)
+
+Amazon Redshift supports multiple Availability Zones (Multi-AZ) deployments for provisioned RA3 clusters. By using Multi-AZ deployments, your Amazon Redshift data warehouse can continue operating in failure scenarios when an unexpected event happens in an Availability Zone.
```

Comment:
Please insert appropriate line breaks. And it would be good to include an explanation of using the `multiAz` property.

Reply:
> And it would be good to include an explanation for the need of the multiAz property.
Does this mean to add a sentence about the benefits of enabling the multiAz property? (Is 'By using Multi-AZ deployments...' not enough for that??)

Reply:
I see! Thanks a lot!